### PR TITLE
LPS-200646 Fix table-layout property to eshow correctly the content o…

### PR DESCRIPTION
**Pull request by sergiojimcos**
**JIRA Related Ticket**
https://liferay.atlassian.net/browse/LPS-200646

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Comment by sergiojimcos** [comment]
ci:test:relevant

File: modules/apps/headless/headless-discovery/headless-discovery-web/src/css/main.css [hunk 1]
```diff
@@ -20,6 +20,10 @@ h4 span {
 	margin-right: 32px;
 }
 
+.swagger-ui .responses-table {
```
Comment (ccorreagg) [inline modules/apps/headless/headless-discovery/headless-discovery-web/src/css/main.css:23]: It is not sorted

**Review by ccorreagg — CHANGES_REQUESTED** [review]

**Comment by yanan** [comment]
Failures in relevant are not related, here are more details on [compare run results](https://testray.liferay.com/home/-/testray/runs/compare?compare=true&view=details&statusA=3&testrayRunIdB=2454607767&testrayRunIdA=2454770529&statusB=2)
- flaky failures because of `type AccountGroup not found in schema`

File: modules/apps/headless/headless-discovery/headless-discovery-web/src/css/main.css (hunk 1 above)
Comment (sergiojimcos) [reply modules/apps/headless/headless-discovery/headless-discovery-web/src/css/main.css:23]: Weird why the SF doesn't complain it.

Comment (sergiojimcos) [reply modules/apps/headless/headless-discovery/headless-discovery-web/src/css/main.css:23]: Thanks @ccorreagg!!

**Comment by sergiojimcos** [comment]
Rebase to Brian here: https://github.com/brianchandotcom/liferay-portal/pull/143465

**Pull request closed without merging**